qemu/tests/virtio_console.py

```diff
@@ -1334,19 +1356,20 @@ def _virtio_dev_add(vm, pci_id, port_id, console="no"):
             port = "console-"
             port_type = "virtconsole"
         port += "%d-%d" % (pci_id, port_id)
-        ret = vm.monitors[0].cmd("device_add %s,"
-                                 "bus=virtio_serial_pci%d.0,"
-                                 "id=%s,"
-                                 "name=%s"
-                                 % (port_type, pci_id, port, port))
+        ndev = qdevices.QDevice(port_type)
+        for key, value in {'id': port, 'name': port, 'bus': "virtio_serial_pci"
+                           "%d.0" % pci_id}.items():
+            ndev.set_param(key, value)
+        (result, ret) = vm.devices.simple_hotplug(ndev, vm.monitor)
```

Comment:
ret is short for result. It seems `ret` here is `verify_hotplug`'s output
Please come up a meaningful name.